docs/release/trg-0/trg-1-07.md

```diff
@@ -0,0 +1,30 @@
+---
+title: TRG 1.07 - End-User Manual (User Assistance)
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 31-Jan-2024  | created initial Draft                  |
+
+## Why
+
+End-User Manual is essential for providing user assistance and ensuring a positive user experience. The user of the application is the receipient of this documentation. Typical contents are screenshots, description of functionality and steps that guide through critical processes.
+
+## Description
+
+- The End-User Manual **has to** be stored in the folder: ```docs/user/```
+- End-User Manual is available with the software at the same time.
+- End-User Manual is correct and up to date; English is a must.
```

Comment:
See my comments on the admin guide. They also apply here.
We don't hand over, we should not claim correctness, we do everything to the best of our knowledge, docs folder in leading repo as `.md`

Reply:
delete line 22